crates/rpc/rpc-engine-api/src/engine_api.rs

```diff
@@ -933,6 +927,20 @@ where
     }
 }
 
+impl<Provider, EngineT: EngineTypes, Pool, Validator, ChainSpec>
+    EngineApi<Provider, EngineT, Pool, Validator, ChainSpec>
+{
+    async fn new_payload_v1_metered(
+        &self,
+        payload: ExecutionPayloadV1,
+    ) -> RpcResult<PayloadStatus> {
+        let start = Instant::now();
+        let res = Self::fork_choice_updated_v3(self, fork_choice_state, payload_attributes).await;
+        self.inner.metrics.latency.fork_choice_updated_v3.record(start.elapsed());
+        self.inner.metrics.fcu_response.update_response_metrics(&res);
+    }
```

Comment:
nice, this can be moved to the existing impl block, next to the new_payload_v1 fn  so that this actually compiles

Reply:
I dont get it sir. I have a lot of errors.

Reply:
![image](https://github.com/user-attachments/assets/a54c4b1f-5148-498e-9d21-c30fc1567767)

Reply:
the existing impl block is up higher in the file

Reply:
Yeah @Rjected and I still have a bunch of errors:

![image](https://github.com/user-attachments/assets/4077d507-a08e-44f6-a95e-6abb04a766ca)


Should I take this:

```rust

async fn new_payload_v1(&self, payload: ExecutionPayloadV1) -> RpcResult<PayloadStatus> {
        trace!(target: "rpc::engine", "Serving engine_newPayloadV1");
        Ok(Self::new_payload_v1_metered(self, payload).await?)
    }

```

And put it on the above impl block of EngineApi?